torch/ao/quantization/fx/_convert_do_not_use.py

```diff
@@ -66,6 +66,114 @@ def insert_dequantize_node(
             if user_node is not dequantize_node:
                 user_node.replace_input_with(node, dequantize_node)
 
+
+def convert_standalone_module(
+        node: Node,
+        modules: Dict[str, torch.nn.Module],
+        model: torch.fx.GraphModule,
+        is_reference: bool,
+        backend_config_dict: Dict[str, Any]):
```

Comment:
can we add a docblock

Reply:
sure, this is added to reduce the complexity of the convert_fx -> _convert_fx_do_not_use PR, will add docs and modify code in that PR so that we can land this faster